src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -421,6 +440,10 @@ else if(isFixed(loansPolicy)) {
     final DateTime systemDate = ClockManager.getClockManager().getDateTime();
 
     final Result<DateTime> recallDueDateResult =
+        loan.isOverdue() &&
+        allowRecallsToExtendOverdueLoans() &&
+        getAlternateRecallReturnInterval() != null ?
```

Comment:
Does this mean that if this code is executed when either the loan isn't overdue or recalls are not allowed to extend the loan then it will still try to extend it by the `recallReturnInterval`?

I think this is the case because of the use of `&&` meaning that if any of them is false, the whole statement will be false.

Reply:
Is there an existing test for an overdue loan where recalls are disallowed from extending overdue loans and there is a recall return interval? If not, please add it.

As I understand it, in that situation the loan should not be extended. I think this code might mean it is.